src/colorizer/ColorizeCanvas.ts

```diff
@@ -155,11 +156,8 @@ export default class ColorizeCanvas implements IRenderCanvas {
   private renderer: WebGLRenderer;
   private pickRenderTarget: WebGLRenderTarget;
 
+  // TODO: Force params to be provided in constructor?
   protected params: RenderCanvasStateParams | null;
-  // Categorical palette is stored separately from params because it's not a
-  // direct part of state, and the ColorRamp class must be disposed and
-  // recreated when the palette changes.
-  protected categoricalPalette: ColorRamp;
```

Comment:
Previously `ColorizeCanvas` had to handle creating and destroying this `ColorRamp` object. I've moved responsibility for that into the store!